File: test/Data.Analyzers.Vsix.Test/VisualBasic/InvalidRegisterMounterInvocation.vb
Public Class InvalidRegisterMounterInvocation
    Inherits Model

    Shared Sub New()
        Dim _column1 = RegisterColumn(Function(x As InvalidRegisterMounterInvocation) x.Column1)
    End Sub

    Private Shared Sub AnotherMethod()
        RegisterColumn(Function(x As InvalidRegisterMounterInvocation) x.Column2)
    End Sub

    Private m_Column1 As _Int32
    Public Property Column1 As _Int32
        Get
            Return m_Column1
        End Get
        Private Set
            m_Column1 = Value
        End Set
    End Property

    Private m_Column2 As _Int32
    Public Property Column2 As _Int32
        Get
            Return m_Column2
        End Get
        Private Set
            m_Column2 = Value
        End Set
    End Property
End Class

File: test/Data.Analyzers.Vsix.Test/CSharp/InvalidRegisterMounterInvocation.cs
namespace DevZest.Data.Analyzers.Vsix.Test.CSharp
{
    public class InvalidRegisterMounterInvocation : Model
    {
        static InvalidRegisterMounterInvocation()
        {
            var _column1 = RegisterColumn((InvalidRegisterMounterInvocation x) => x.Column1);
        }

        private static void AnotherMethod()
        {
            RegisterColumn((InvalidRegisterMounterInvocation x) => x.Column2);
        }

        public _Int32 Column1 { get; private set; }

        public _Int32 Column2 { get; private set; }
    }
}
